.github/workflows/release.yml

```diff
@@ -22,6 +22,22 @@ jobs:
       - name: Configure git identity
         run: |
             git config --global user.name "Release Workflow"
+      - name: create a new branch that references commit id
+        run: git checkout -b ${{ github.event.inputs.version_number }} ${{ github.event.inputs.commit_id }}
+      - name: Generate SBOM
+        uses: FreeRTOS/CI-CD-Github-Actions/sbom-generator@main
+        with:
+          repo_path: ./
+          source_path: ./source
+          url: 'https://github.com/FreeRTOS/corePKCS11'
+      - name: Configure git identity
+        run: |
+            git config --global user.name "Release Workflow"
```

Comment:
name and email might be able to be configured by environment variables. It should be able to use the name/email of the account that triggered the release. Though that is not required

If name is set, email should be too.

Reply:
Fixed